### PR TITLE
Update LpHarvester.sol

### PR DESCRIPTION
**Pull request by soliditycrew**
Make the contract manageable through the Keep3r Network, to encourage a trustless Vault. A basic proof of concept...
Required:
Logic for workable boolean 
Check the IUniswapRouter interface is inheriting the swapExactTokensForTokens() function for _swap()
I may be missing somethings, this is just an initial commit.
Let me know what you guys think about this idea anyway, I won't continue with the development if it's not something you would look to merge?

Thanks,

### CODE REVIEW

**Comment by soliditycrew** [comment]
If the idea of the Keep3r Network managing the LpHarvester doesn't take off, I still feel as though the harvestVault() function should be somewhat trustless, maybe take the idea of the minKeep condition used in upkeep(), where functions can be called by any EOA or Contract as long as they meet the conditions of the modifier. The modifier could have a minINV() (Minimum INV) where callers of the function must hold a minimum amount of INV for the tnx to be succeed.

File: contracts/LpHarvester.sol [hunk 1]
```diff
@@ -34,6 +103,42 @@ contract LpUniswapHarvester is Ownable {
...
+    }
+    
+   
+ /* Needs logic to check if the Vault rewards can be harvested else return false  */
+    function workable(address harvester) public view returns (bool) {
+       for () { 
+            if () { 
+                return true;
+            }
+        return false;
```
Comment (fqlx) [inline contracts/LpHarvester.sol:124]: Just 1 line this to `return false`

**Comment by soliditycrew** [comment]
@fqlx - Thanks for reviewing, committed the change.

**Comment by nourharidy** [comment]
Thanks you for the PR. I'm very excited to have Kp3r integration.

LP Harvester is deprecated and was part of the plan for the original token launch.
We need to move these changes to the UniswapHarvester.sol instead

**Comment by soliditycrew** [comment]
_Closed and will work on UniswapHarvester.sol instead. Thanks_

**Pull request closed without merging**